frame/staking/src/mock.rs

```diff
@@ -598,3 +599,21 @@ pub fn make_all_reward_payment(era: EraIndex) {
 		assert_ok!(Staking::payout_validator(Origin::signed(validator_controller), era));
 	}
 }
+
+/// FIXME
+/// Approximate value for the reward tests at the beginning blocks.
+pub trait Approx {
+	fn approx(self) -> Balance
+	where
+		Self: AtLeast32Bit,
+	{
+		let v: u128 = self.try_into().unwrap_or(0);
+		if v > APPROXIMATE ^ 2 {
+			v.checked_div(APPROXIMATE).unwrap_or(v)
+		} else {
+			v
+		}
+	}
+}
+
+impl Approx for u128 {}
```

Comment:
This `trait` terms to support fault-tolerance for two calculating way:

+ calculate with `Perbill`
+ calculate with approx values directly